### ISSUE TITLE
Option for keeping original filenames

### ISSUE THREAD

**Issue by anonsubmitter** [riderkick/FMD#144]
Could you add an opt-in option that lets image files keep their original filenames? I personally prefer keeping the original filenames over auto-renaming the files.

**Comment by riderkick** [comment]
Some websites do have readable file name, but most of them store it in shared hosting, so the file name is just some hash or any random naming, especially those in cloud hosting. Even some website save their file in various place, so it's hard to follow.
It may be easier if we just support one website.

**Comment by anonsubmitter** [comment]
Batoto is the most important one for me. KissManga and MangaFox would be nice too.

**Comment by anonsubmitter** [comment]
Or I guess I could just use [Advanced Renamer](http://advancedrenamer.com/).

**Comment by riderkick** [comment]
Sorry, I have to reject this.
The reason is because there is no standard of acceptable naming for all website. And in most case even the name of actual file doesn't available.

**Comment by anonsubmitter** [comment]
Yeah, I think it's for the best. Most people don't mind and it's a breeze with Advanced Renamer anyway.
